packages/ui/src/preset-item.tsx

```diff
@@ -79,41 +45,45 @@ export class PresetItem {
     });
   }
 
-  get playerConfig(): KalturaPlayerPresetComponent | null {
-    const containerName = getPlayerPresetContainer(
-      this._options.data.container
-    );
-    const {relativeTo} = this._options.data;
+  get playerConfig(): KalturaPlayerPresetComponent[] {
+    const configs: KalturaPlayerPresetComponent[] = [];
 
-    if (!containerName) {
-      this._logger.warn(`unknown container requested`, {
-        method: 'playerConfig',
-      });
-      return null;
-    }
+    for (const presetType in this._options.data.presetAreas) {
+      const presetContainer = this._options.data.presetAreas[presetType];
+      const {relativeTo} = this._options.data;
 
-    const result: KalturaPlayerPresetComponent = {
-      label: this._options.data.label,
-      presets: this._options.data.presets,
-      container: containerName,
-      get: this._render,
-    };
-
-    if (relativeTo) {
-      switch (relativeTo.type) {
-        case RelativeToTypes.After:
-          result['afterComponent'] = relativeTo.name;
-          break;
-        case RelativeToTypes.Before:
-          result['beforeComponent'] = relativeTo.name;
-          break;
-        case RelativeToTypes.Replace:
-          result['replaceComponent'] = relativeTo.name;
-          break;
+      if (!presetContainer) {
```

Comment:
I'm not sure if we need to warn about this one, as it is only relevant if someone passed an empty value. anyway the message should be self explained. something like `Cannot register component to core player using the preset manager for preset ${presetType}. preset area name is missing. Ignoring this request.`

Reply:
done